webapp/jira.py

```diff
@@ -57,8 +55,11 @@ def __request__(
             params=params,
         )
 
-        if response.status_code == 200 or response.status_code == 201:
-            return response.json()
+        if response.status_code in [200, 201, 204]:
+            try:
+                return response.json()
+            except json.JSONDecodeError:
+                return response.status_code
```

Comment:
We shouldn't return a status code instead of json, as the interface should be consistent. Also, an error in the json should in fact return a 500 error

Reply:
Actuall the error is not in json exactly, sometimes the body is not a "JSON" response and sometimes there is no response at all.
I will correct the status code thing though.